### PR TITLE
MF-467 - Make CoAP ping period configurable

### LINKED ISSUE #467: Make CoAP ping period configurable
**FEATURE REQUEST**

1. Is there an open issue addressing this request? If it does, please add a "+1" reaction to the
   existing issue, otherwise proceed to step 2.

2. Describe the feature you are requesting, as well as the possible use case(s) for it.
Make CoAP adapter 'ping period' configurable. According to [RFC 7641](https://tools.ietf.org/html/rfc7641#page-18), `CONF` notification must be sent by the server at least every 24 hours, so it makes sense to make this period configurable. Valid values are between 1 and 24. 

3. Indicate the importance of this feature to you (must-have, should-have, nice-to-have).
This is a should-have feature.

### PR DESCRIPTION
**Pull request by dborovcanin**
Signed-off-by: Dusan Borovcanin <dusan.borovcanin@mainflux.com>

### What does this do?
This PR makes CoAP adapter ping period configurable.

### Which issue(s) does this PR fix/relate to?
Closes #467.

### Have you included tests for your changes?
No. Test will be included as a part of the Adapter tests.

### Did you document any new/modified functionality?
Yes.

### CODE REVIEW

**Review by drasko — DISMISSED** [review]
LGTM

File: cmd/coap/main.go [hunk 1]
```diff
@@ -117,13 +121,23 @@ func loadConfig() config {
 		log.Fatalf("Invalid value passed for %s\n", envClientTLS)
 	}
 
+	pp, err := strconv.ParseInt(mainflux.Env(envPingPeriod, defPingPeriod), 10, 64)
+	if err != nil {
+		log.Fatalf("Invalid value passed for %s\n", envPingPeriod)
```
Comment (anovakovic01) [inline cmd/coap/main.go:126]: Why not using mainflux logger?

File: cmd/coap/main.go [hunk 2]
```diff
@@ -117,13 +121,23 @@ func loadConfig() config {
...
+	}
+
+	if pp < 1 || pp > 24 {
+		log.Fatalf("Value of %s must be between 1 and 24", envPingPeriod)
```
Comment (anovakovic01) [inline cmd/coap/main.go:130]: Same here.

**Review by anovakovic01 — CHANGES_REQUESTED** [review]

**Review by anovakovic01 — APPROVED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**